requirements/base.in

```diff
@@ -2,5 +2,5 @@
 
 -c constraints.txt
 
-Django>=1.8,<2.0          # Web application framework
-
+Django>=1.11          # Web application framework
+Django<2.0; python_version < "3"    # Web application framework
```

Comment:
As I've noted in some other PRs, I think we need to omit this line; the `python_version` constraint gets inappropriately applied to the Django dependency as a whole in the files generated by `pip-compile`.